File: Swift/delete-columns-to-make-sorted.swift
class Solution {
    func minDeletionSize(_ A: [String]) -> Int {
        var res = 0
        var charsArr = A.map { Array($0) }

        for c in 0..<charsArr[0].count {
            for r in 1..<charsArr.count {
                if charsArr[r-1][c] > charsArr[r][c] {
                    res += 1
                    break
                }
            }
        }

        return res
    }
}

File: Python/delete-columns-to-make-sorted.py
class Solution:
    def minDeletionSize(self, A):
        """
        :type A: List[str]
        :rtype: int
        """
        res = 0

        for c in range(len(A[0])):
            for r in range(1, len(A)):
                if A[r-1][c] > A[r][c]:
                    res += 1
                    break

        return res
